analysers/analyser_merge_charging_station_FR.py

```diff
@@ -22,9 +22,87 @@
 
 from modules.OsmoseTranslation import T_
 from .Analyser_Merge import Analyser_Merge_Point, Source, CSV, Load_XY, Conflate, Select, Mapping
+import re
 
 
 class Analyser_Merge_Charging_station_FR(Analyser_Merge_Point):
+    def remove_trailing_zeros(input_string):
+        """
+        Removes dot and zeros at the end of a floating number typed in string
+        """
+        return str(input_string).replace('.00', '').replace('.0', '')
```

Comment:
This would fail in case there's 3 zeros (not sure if that is ever going to be likely in the external data source). This would be more fail safe.
`return input_string if not "." in input_string else input_string.rstrip("0.") or "0"`
There might be better ways though, as I think it's only floats (as string) that will be checked?

Reply:
added your contribution to the source, and a test to confirm that it works fine